javascript/javascript1/week3/phoneusage.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>
+            smartphone-usage
+        </title>
+        <meta charset="utf-8">
+             
+    </head>
+    <body>
+    
+    <script>
+    //smartphone usage
+//adding activities on the phone with todays's date.
+"use strict"
+const activities = [];
+function addActivity(activity, duration){
+  
+    let date= new Date();
+    let todaysDate =date.toLocaleDateString();
+  activities.push({date:todaysDate, activity:activity, duration:duration}); 
+    
+}
+addActivity( "Youtube", 30);
+addActivity( "Faceb00k", 10);
+addActivity( "Whatsapp", 20);
+addActivity( "Instagram", 20);
+addActivity("News", 10)
+console.log(activities)
+/*[{date: "11/11/2020", activity: "Youtube", duration: 30},
+{date: "11/11/2020", activity: "Faceb00k", duration: 10},
+{date: "11/11/2020", activity: "Whatsapp", duration: 20},
+{date: "11/11/2020", activity: "Instagram", duration: 20},
+{date: "11/11/2020", activity: "News", duration: 10}*/
+
+//showStatus(activities); setting a limit for smartphone usage
+
+let totalTime = 0;
+const usageLimit = 120;
+function showStatus(activities){
+  if(activities.length===0){
+    console.log("Add someactivities before calling showStatus.")
+  }else {
```

Comment:
do you need else?

Reply:
Ya...true, i dont need else here, removed it..tq